### ISSUE TITLE
Odd characters in release zip files, but not in sources...

### ISSUE THREAD

**Issue by gaiaclary** [KhronosGroup/OpenCOLLADA#563]
Hi,

There seems to be something odd going on with creating the release zip files. The problem is with non ascii characters in comments. Here is an example:

- Download: https://github.com/KhronosGroup/OpenCOLLADA/archive/v1.6.51.zip
- then open the file COLLADAFWEffect.h
- and inspect line 37 
- compare this line with what is in master.

Here is the difference (yes the 2 strings are really different :)

underlying library as the “FX Runtime”.   (from the zip file)
underlying library as the "FX Runtime".   (from master)

Check the first double quote. That is a non ascii character in the zip file, while in the master it is a simple double quote. You might think this is nothing serious. However the non ascii characters in the comments seem to make trouble. When we build Blender then we get more than 200 utf-8 warnings from the OpenCollada header files...

We propose that this gets fixed for future release zip files (replace the non ascii doublequotes by ascii quotes) ? Is that feasible and doable?

cheers,
Gaia

**Comment by gaiaclary** [comment]
Ah, now i understand.

I looked at the sources with git blame and found that something was changed in mid 2016. However the zip file was from mid 2017 so i assumed this change must have already been included in the zip and something went wrong when the zip was created from the sources. But just now i see the patch was merged after the zip file was created.  That might explain it now :)

I will prepare to update to a newer version of the OpenCollada library then.

cheers,
Gaia
